django/thunderstore/repository/admin/package_version.py

```diff
@@ -17,11 +22,33 @@ def extract_file_list(modeladmin, request, queryset: QuerySet):
 extract_file_list.short_description = "Queue file list extraction"
 
 
+@transaction.atomic
+def reject_version(modeladmin, request, queryset: QuerySet[PackageVersion]):
+    for version in queryset:
+        version.review_status = PackageVersionReviewStatus.rejected
+        version.save(update_fields=("review_status",))
+
+
+reject_version.short_description = "Reject"
+
+
+@transaction.atomic
+def approve_version(modeladmin, request, queryset: QuerySet[PackageVersion]):
+    for version in queryset:
+        version.review_status = PackageVersionReviewStatus.approved
+        version.save(update_fields=("review_status",))
+
+
+approve_version.short_description = "Approve"
```

Comment:
Did you consider if these should call the approve/reject actions that fire audit log events etc?

I'm aware there are other similar call sites in the codebase, but I suspect it would be better to e.g. have a `mass_reject`/`mass_approve` function for the model itself to handle the implementation of to keep the locations with write access to review statuses minimal. Perhaps eventually even that could be turned into a more generic mixin